File: ios/RNSessionManager.m
//
//  SessionModule.m
//  HotwireRNDemo
//
//  Created by Bartłomiej Fryz on 09/08/2022.
//

#import <React/RCTViewManager.h>
#import <React/RCTEventEmitter.h>
#import <React/RCTUIManager.h>
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(RNSessionManager, NSObject)

  RCT_EXPORT_VIEW_PROPERTY(onMessage, RCTDirectEventBlock)

@end

File: old/ios/RNSessionManager.swift
//
//  SessionModule.swift
//  HotwireRNDemo
//
//  Created by Bartłomiej Fryz on 09/08/2022.
//

import Foundation
import Turbo

@objc(RNSessionManager)
class RNSessionManager: RCTViewManager {
  
  override func view() -> UIView! {
    return RNSession()  
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
}
